### ISSUE TITLE
TechDraw: "No default template found" error in weekly

### ISSUE THREAD

**Issue by FEA-eng** [FreeCAD/FreeCAD#13539]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I'm unable to use TechDraw in the newest Windows weekly. When I click the _Insert Default Page_ button, I get the "No default template found" error and the following warning message appears in the report view:

`15:16:36  Template File: E:\Program Files\FreeCAD_weekly-builds-36874-2024-04-17-conda-Windows-x86_64-py311\data\Mod/TechDraw/Templates/A4_LandscapeTD.svg is not readable`

@WandererFan FYI

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36874 (Git)
Build type: Release
Branch: main
Hash: a4dd600ec6d5a53a9960cd9f8693d7d307b9e359
Python 3.11.8, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: Polish/Poland (pl_PL)
```

### Subproject(s) affected?

Techdraw

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dplanella** [comment]
I'm not sure if this is related, but on https://github.com/FreeCAD/FreeCAD/pull/13368 I renamed the custom `TD` and `m52` TechDraw templates to include an underscore for consistency. I could imagine that if you had your default template set to any of those, you'd run into this error.

**Comment by WandererFan** [comment]
I struggle with windows issues, but is the presence of both forward and backward slashes in the file path relevant?

What is in your preference for the default template?

**Comment by FEA-eng** [comment]
The paths are empty:

![templates](https://github.com/FreeCAD/FreeCAD/assets/59876896/240f4675-f2a2-4542-a4a0-58cf2c3b6884)

It was working normally in the previous weekly builds.

**Comment by maxwxyz** [comment]
can confirm, same behavior.
Discussion: https://forum.freecad.org/viewtopic.php?p=756351
```
OS: Windows 11 build 22631
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36729 (Git)
Build type: Release
Branch: main
Hash: 6ca35709ddc7f95fb84eef4c24973dc489e5acde
Python 3.11.8, Qt 5.15.8, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
Installed mods: 
  * BIM 2021.12.0
  * CfdOF 1.25.11
  * Curves 0.6.35
  * dodo 1.0.1
  * fasteners 0.5.20
  * fcVM-WB
  * freecad.gears 1.2.0
  * OpenTheme 2024.4.20
  * sheetmetal 0.4.13
```

**Comment by maxwxyz** [comment]
![image](https://github.com/FreeCAD/FreeCAD/assets/6246609/597858c0-6aac-442c-9376-9212f46f2db0)

**Comment by prokoudine** [comment]
It's the same on Linux (Ubuntu 23.10).

**Comment by WandererFan** [comment]
fixed by https://github.com/FreeCAD/FreeCAD/pull/13784

**Comment by FEA-eng** [comment]
> fixed by #13784

Yes, but the default template is a blank page now so it’s kind of a regression. Do you plan to fix that? Should I create a new issue?
